doc/changelog/12-misc/10019-PG-proof-diffs.rst

```diff
@@ -0,0 +1,5 @@
+- Proof General can now display Coq-generated diffs between proof steps
+  in color.  (Updated the behavior of the -emacs command line option to
+  enable this.)
```

Comment:
I think this parenthesis is not interesting to the end user and could be removed.